apps/doc/src/app/components/file-upload/examples/i18n/file-upload-i18n-example.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { PRIZM_LANGUAGE, PRIZM_RUSSIAN_LANGUAGE, PrizmLanguageFileUpload } from '@prizm-ui/i18n';
+import { of } from 'rxjs';
+
+export const PRIZM_ENGLISH_FILE_UPLOAD: PrizmLanguageFileUpload = {
+  fileUpload: {
+    dropzone__description: 'Select a file or drag it to this area',
+    dropzone__title: 'File upload',
+    btn__select: 'Browse',
+  },
+};
+
+@Component({
+  selector: 'prizm-file-upload-i18n-example',
+  templateUrl: './file-upload-i18n-example.component.html',
+  providers: [
+    {
+      provide: PRIZM_LANGUAGE,
```

Comment:
{
provide: PRIZM_FILE_UPLOAD,
useFactory(() => of(PRIZM_ENGLISH_FILE_UPLOAD))
}